pwiz_tools/Skyline/Model/Results/ChromHeaderInfo.cs

```diff
@@ -1029,7 +1029,7 @@ public struct ChromPeak : ISummaryPeakData
         private readonly float _retentionTime;
         private readonly float _startTime;
         private readonly float _endTime;
-        private readonly float _area;
+        private float _area;

```

Comment:
I think more than just the area needs to change. Background needs to become zero. Height and area should become equal. FWHM also needs to become zero. Mass error needs to become the mass error at the single point, points across peak should be 1. And the RT needs to become the RT of the single spectrum for all transitions, not the time at which each has its maximum intensity within start-end time.